dmv/dmv.js

```diff
@@ -19,13 +19,18 @@ DMV.prototype.currentCustomerFor = function(agent) {
 }
 
 DMV.prototype.nextCustomer = function() {
+  if (this.customers.length === 0) {
+    throw new Error('No customers')
+  }
   const currentCustomer = this.customers.shift()
-  this.agentList.find(agent => {
+  let currentAgent = null;
+  this.agentList.forEach(agent => {
     if (!this.attending[agent]) {
       this.attending[agent] = currentCustomer
-      return true
+      currentAgent = agent
     }
   })
+  return {agent: currentAgent, customer: currentCustomer}
 }
 
```

Comment:
Reading through the test and pulling out the methods that were needed in order to pass is important. I found that creating a list of all the different Methods helped. Also defining the resolve method!!! I think the naming for the parameters was also well done, it made the code clear and concise/easily readable.